src/lib.rs

```diff
@@ -11,7 +11,7 @@ pub mod hash;
 pub mod keys;
 pub mod musig;
 pub mod range_proof;
-pub mod signatures;
+pub mod signature;
```

Comment:
These two mods with the same name is going to be confusing

Reply:
Trying to keep the old one not to break every program that uses tari_crypto. 
Changed the name to: `signature_schemes`

Reply:
I saw the comment below, changing to just signatures, removing the old use.